tests/jeos/firstrun.pm

```diff
@@ -109,7 +109,7 @@ sub verify_bsc {
 sub verify_partition_label {
     my $label = 'gpt';
 
-    if (is_s390x || (is_aarch64 && get_var('HDD_1') =~ /.*raw\.xz$/ && !is_community_jeos)) {
+    if (is_s390x || check_var('BACKEND', 'generalhw') || (is_aarch64 && get_var('HDD_1') =~ /.*raw\.xz$/ && !is_community_jeos)) {
```

Comment:
There is no connection from `check_var('BACKEND', 'generalhw')` to "uses MBR". What about non RPi?